### PR TITLE
isce2

### PR DESCRIPTION
**Pull request by piyushrpt**
Checklist

- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/isce2/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (chrisburr) [inline recipes/isce2/LICENSE]: This file can be removed as it's included in the source tarball.

File: recipes/isce2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      - ISCEHOME=`$PYTHON -c "import isce, os; print(os.path.dirname(isce.__file__))" | tail -n 1`  # [unix]
+      - ${ISCEHOME}/applications/topsApp.py --help --steps  # [unix]
+      - ${ISCEHOME}/applications/stripmapApp.py --help --steps  # [unix]
+      - python {{ RECIPE_DIR }}/run_test.py  # [unix]
+
+about:
+  home: http://github.com/isce-framework/isce2
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```
Comment (chrisburr) [inline recipes/isce2/meta.yaml]:
```suggestion
  license_file: LICENSE-2.0.txt
```

File: recipes/isce2/build.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+mkdir $SRC_DIR/{build,install,config,bin}
+
+##Setup a link to cython here
+ln -s $PREFIX/bin/cython $SRC_DIR/bin/cython3
+
+#Setting up some environment variables
+export PATH=$PATH:$PREFIX/bin:$SRC_DIR/bin
```
Comment (chrisburr) [inline recipes/isce2/build.sh]: `${PREFIX}/bin` should already be in the `PATH`:
```suggestion
export PATH="${PATH}:${SRC_DIR}/bin"
```

**Comment by piyushrpt** [comment]
@chrisburr Thanks for the suggestions and fixing issues with recipe. Do let me know, if you would like to see any more changes made or things fixed.

**Pull request merged**